CodeStarterCamp_Week5/AuditionManager.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  AuditionManager.swift
+//  CodeStarterCamp_Week5
+//
+//  Created by 김형철 on 2023/03/22.
+//
+
+import Foundation
+
+struct AuditionManager {
+    var totalApplicantsList: [Person]
+    var passedApplicantsList: [Person] = [Person]()
```

Comment:
```suggestion
    var passedApplicantsList: [Person] = []
```

빈 배열일 경우 간단히 `[]`만 사용하는 것이 더 일반적인 것 같아요